rest-server/src/components/auth/authController.ts

```diff
@@ -1,45 +1,46 @@
-// import { Request, Response, NextFunction } from 'express';
+import { Request, Response, NextFunction } from 'express';
 import * as bcrypt from 'bcrypt';
 import * as jwt from 'jsonwebtoken';
 
+import db from '../../config/db';
 import {
-  findUser,
-  saveUser
-} from '../user/userQueries';
+  getUserHelper,
+  postUserHelper
+} from '../user/userSQLHelpers';
 import { error } from '../../lib/log';
 
 export const createTokenForUser = async (userEmail: string) => {
   return await jwt.sign({ sub: userEmail, iat: Math.floor(Date.now() / 100) }, process.env.JWT_SECRET, { expiresIn: '7d'} );
-}
+};
 
-export const login = async (req: any, res: any, next: any) => {
-// export const login = async (req: Request, res: Response, next: NextFunction) => {
+export const login = async (req: Request, res: Response, next: NextFunction) => {
+  const { email } = req.body;
   try {
-    const token = await createTokenForUser(req.user[0].email);
-    return res.status(200).append('authorization', token).send('successfully logged in')
+    const token = await createTokenForUser(email);
+    return res.status(200).set('authorization', token).send(email);
```

Comment:
I changed it because I'm pretty sure `.set()` will allow us to pluck it off the headers

Reply:
Ok that’s cool If it lets us pluck it off headers, so the only thing now is instead of sending just the email back on login we should send the user profile .... agree or disagree with front load strategy? .... I think that’s better instead of doing on init or some
Shit

Reply:
yeah I can do that. Just the user info or all the info pertaining to them?

Reply:
All the info pertaining to them ideally is what we want. If for some reason it
Becomes too big we can choose what to pull out of it.

Reply:
True, it's gonna be a query that will grow then. Needs a lot of joins from tables I haven't created yet. So for now, it'll just return the user profile and the projects associated with them

Reply:
alright sounds good, we'll just keep it as efficient as possible .... so we'll only return shit we need.